components/higher-order/with-global-events/index.js

```diff
@@ -55,7 +55,12 @@ function withGlobalEvents( eventTypesToHandlers ) {
 
 			handleRef( el ) {
 				this.wrappedRef = el;
-				this.props.forwardedRef( el );
+				// Any component using `withGlobalEvents` that is not setting a `ref`
+				// will cause `this.props.forwardedRef` to be `null`, so we need this
+				// check.
+				if ( this.props.forwardedRef ) {
```

Comment:
This sort of makes sense. @nerrad, how do you handle this in your PR?

Reply:
Well the pr is simply passing through `forwardedRef` to the components.  I think HOC's will still need to handle the `forwardedRef` similarly (depending on how ref is used within the component the HOC is wrapping).

Reply:
So to be clear, _currently_ the `createHigherOrderComponent` is just passing along `forwardedRef` through the HOC to the WrappedComponent.  This means, that it's still possible that `forwardedRef` is null.  What I could maybe do is set `forwardedRef` to `noop` when `ref` is null?